CIP-0001/README.md

```diff
@@ -120,12 +122,18 @@ Authors:
     - Duncan Coutts <duncan.coutts@iohk.io>
 Implementors: N/A
 Discussions:
-    - https://github.com/cardano-foundation/cips/pulls/1
+    - https://github.com/cardano-foundation/cips/pull/366
```

Comment:
```suggestion
    - https://github.com/cardano-foundation/CIPs/pull/366
```
capitalisation, it doesnt really matter of course

Reply:
good spot... I'll fix the non-canonical names in the header as well.